libass/ass_bitmap.c

```diff
@@ -30,31 +30,38 @@
 #include FT_GLYPH_H
 #include FT_OUTLINE_H
 
+#include "ass_cpu.h"
 #include "ass_utils.h"
 #include "ass_outline.h"
 #include "ass_bitmap.h"
 #include "ass_render.h"
 
 
 #define ALIGN           C_ALIGN_ORDER
-#define DECORATE(func)  ass_##func##_c
```

Comment:
Do we really need this change? It adds a couple of lines but not really helps that much.